cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-actions-manu-is-hidden.cy.js

```diff
@@ -0,0 +1,70 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+
+let user;
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.bulkEditCsvView.gui,
+        Permissions.bulkEditCsvEdit.gui,
+        Permissions.bulkEditLogsView.gui,
+        Permissions.uiUserEdit.gui,
+      ]).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.bulkEditPath,
+          waiter: BulkEditSearchPane.waitLoading,
+        });
+
+        FileManager.createFile(`cypress/fixtures/${userBarcodesFileName}`, user.barcode);
+      });
+    });
+
+    after('Delete test data', () => {
+      Users.deleteViaApi(user.userId);
+      FileManager.deleteFile(`cypress/fixtures/${userBarcodesFileName}`);
+    });
+
+    it(
+      'C367997 Verify that "Actions" menu is hidden on the "Logs" tab-- Local approach (firebird) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+      () => {
+        BulkEditSearchPane.verifySpecificTabHighlighted('Identifier');
+        BulkEditSearchPane.verifyPanesBeforeImport();
+
+        BulkEditSearchPane.verifyRecordIdentifierItems();
+        BulkEditSearchPane.verifyBulkEditPaneItems();
+        BulkEditSearchPane.actionsIsAbsent();
```

Comment:
Please, ask QA fix TC if
"Actions" menu is enabled at the right upper corner
is wrong assertion